src/app/components/common/node-editor/node-editor.component.ts

```diff
@@ -668,9 +668,9 @@ export class NodeEditorComponent implements OnInit {
     }
   }
 
-  deleteService() {
-    this.servicesApiService.deleteService(this.deleteServiceName)
-      .subscribe((data: any) => {
+  async deleteService() {
+    await this.servicesApiService.deleteService(this.deleteServiceName)
+      .then((data: any) => {
```

Comment:
await with then?  mixing 2 same things together. 
> Async/Await is a syntactic sugar for promises, a wrapper making the code execute more synchronously

> In Promise, then method called when the promise becomes fulfilled. 

We should either use async/await or Promise.then not mixing the same things. Also not sure do we really need this change?